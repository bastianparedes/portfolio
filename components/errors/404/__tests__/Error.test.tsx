import React from 'react'
import Error from '..'
import { render } from '@testing-library/react'

describe('<Error />', () => {
  it('should render', () => {
    const { container } = render(<Error />)
    expect(container).toMatchSnapshot()
  })
})
