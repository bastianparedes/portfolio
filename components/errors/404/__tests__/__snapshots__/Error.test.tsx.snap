// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Esta página no está disponible.
    </h1>
    <h2>
      Prueba a realizar otra búsqueda.
    </h2>
  </div>
</div>
`;
